### PR TITLE
Add Devnet support

### PR DESCRIPTION
**Pull request by intelliot**
### Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Before / After

Adds support for Devnet, the relatively new XRP Ledger test network.

### CODE REVIEW

File: src/setup/index.ts [hunk 1]
```diff
@@ -36,27 +36,32 @@ const cwd = process.cwd();
     accountInfo = await got.post('https://faucet.altnet.rippletest.net/accounts', {json: true})
       .then(({body}) => body.account)
       .catch((err) => console.log('  ' + err.message));
+  } else if (serverUrl === 'wss://s.devnet.rippletest.net:51233') {
+    console.log('▲ Generating Devnet account credentials...');
+    accountInfo = await got.post('https://faucet.devnet.rippletest.net/accounts', {json: true})
+      .then(({body}) => body.account)
+      .catch((err) => console.log('  ' + err.message));
```
Comment (hbergren) [inline src/setup/index.ts:43]: Is it worth pulling this `await.then.catch` block into a function where you pass in `devnet` | `altnet`?

File: src/setup/index.ts [hunk 2]
```diff
@@ -30,18 +30,18 @@ const cwd = process.cwd();
...
-  } else if (serverUrl === 'wss://s.devnet.rippletest.net:51233') {
-    console.log('▲ Generating Devnet account credentials...');
-    accountInfo = await got.post('https://faucet.devnet.rippletest.net/accounts', {json: true})
+  const network = serverUrl === 'wss://s.altnet.rippletest.net:51233' ? 'altnet' : serverUrl === 'wss://s.devnet.rippletest.net:51233' ? 'devnet' : 'other';
+  async function generateCredentials(network: 'altnet' | 'devnet'): Promise<{
+    address: string,
+    secret: string
+  }> {
+    console.log(`▲ Generating ${network === 'altnet' ? 'Testnet' : 'Devnet'} account credentials...`);
+    return got.post(`https://faucet.${network}.rippletest.net/accounts`, {json: true})
```
Comment (hbergren) [inline src/setup/index.ts:39]: I like the `generateCredentials` function!

File: src/setup/index.ts [hunk 3]
```diff
@@ -30,18 +30,18 @@ const cwd = process.cwd();
...
-  let accountInfo = null;
-  if (serverUrl === 'wss://s.altnet.rippletest.net:51233') {
-    console.log('▲ Generating Testnet account credentials...');
-    accountInfo = await got.post('https://faucet.altnet.rippletest.net/accounts', {json: true})
-      .then(({body}) => body.account)
-      .catch((err) => console.log('  ' + err.message));
-  } else if (serverUrl === 'wss://s.devnet.rippletest.net:51233') {
-    console.log('▲ Generating Devnet account credentials...');
-    accountInfo = await got.post('https://faucet.devnet.rippletest.net/accounts', {json: true})
+  const network = serverUrl === 'wss://s.altnet.rippletest.net:51233' ? 'altnet' : serverUrl === 'wss://s.devnet.rippletest.net:51233' ? 'devnet' : 'other';
```
Comment (hbergren) [inline src/setup/index.ts:33]: This is a bit confusing to me. I get what's going on, but it's a bit hard to parse with two different ternarys happening in the same line.

Comment (intelliot) [reply src/setup/index.ts:33]: So would _if - else if - else_ be better?
